admin-dev/themes/new-theme/js/vue/components/Modal.vue

```diff
@@ -122,24 +125,15 @@
       },
       cancelLabel: {
         type: String,
-        required: false,
-        default() {
```

Comment:
So that's why you had to specify all the missing labels manually in other components. But why do this, the default feature does not exist anymore?